app/legacy_lib/query_donations.rb

```diff
@@ -95,4 +101,11 @@ def self.get_first_payment_for_donation(table_selector = 'donations')
       .order_by('payments.created_at ASC')
       .limit(1)
   end
+
+  def self.update_amount_with_currency(query_row, currency)
+    if query_row[1].to_s.downcase != 'amount'
```

Comment:
Please add a comment before this line explaining that you're doing this to skip the header row. It makes sense to do it but I don't think it's obvious just from looking at it.